flask_appbuilder/security/sqla/manager.py

```diff
@@ -357,7 +357,7 @@ def del_permission_view_menu(self, permission_name, view_menu_name):
 
     def exist_permission_on_views(self, lst, item):
         for i in lst:
-            if i.permission.name == item:
+            if i.permission and i.permission.name == item:
```

Comment:
This is the bugfix. In my specific case `i.permission` was == None and `.name` was failing. I'm unclear on why a None permission made it here. Perhaps some incomplete deletion/cleanup?